safesocket/safesocket.go

```diff
@@ -29,9 +33,54 @@ func ConnCloseWrite(c net.Conn) error {
 	return c.(closeable).CloseWrite()
 }
 
+var processStartTime = time.Now()
+
+// tailscaledStillStarting reports whether tailscaled is probably
```

Comment:
This adds tailscale specific code in an otherwise generic library.

Reply:
Do you propose we create `tailscale.com/safesocket/tailscalesafesocket` ?

Reply:
(That was a joke.)

Look at `safesocket_darwin.go` to convince yourself that safesocket is Tailscale-specific already. :)

Reply:
Lol well ok then